lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1623,34 +1623,34 @@ fc_extras
         # Get units
         attr_units = get_attr_units(cf_coord_var, attributes)
 
-        def cf_var_as_biggus(cf_var):
+        def cf_var_as_array(cf_var):
```

Comment:
Is it worth reconsidering the name here ?
I'd say that "array" sounds too much like numpy + should be replaced by "lazy" -- where before we had "biggus".